### ISSUE TITLE
Missing ApiError export

### ISSUE THREAD

**Issue by canadaduane** [qdrant/qdrant-js#21]
When running code such as the following, I don't know how to tell if the cause of the error is an ApiError, or a more general error, because I can't seem to find ApiError exported by `qdrant-js`:

```js
let collection
try {
 collection = await client.getCollection("bom-ada-002");
} catch(err) {
  if (err instanceof ApiError) {
    // ...
  }
}
```

Am I missing it, or does it need to be exported?

Edit to add: Here is the error I'm seeing in console when I try to `get` a collection that doesn't exist yet:

```
file:///home/duane/tmp/try-qdrant/node_modules/.pnpm/@qdrant+openapi-typescript-fetch@1.2.0/node_modules/@qdrant/openapi-typescript-fetch/dist/esm/fetcher.js:147
                throw new fun.Error(err);
                      ^

ApiError: Not Found
    at Object.fun [as getCollection] (file:///home/duane/tmp/try-qdrant/node_modules/.pnpm/@qdrant+openapi-typescript-fetch@1.2.0/node_modules/@qdrant/openapi-typescript-fetch/dist/esm/fetcher.js:147:23)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async QdrantClient.getCollection (file:///home/duane/tmp/try-qdrant/node_modules/.pnpm/@qdrant+js-client-rest@1.1.6_typescript@5.0.4/node_modules/@qdrant/js-client-rest/dist/esm/qdrant-client.js:633:26) {
  headers: Headers {
    [Symbol(query)]: [
```

**Comment by Rendez** [comment]
This is how you do error handling: https://github.com/qdrant/openapi-typescript-fetch#typed-error-handling I'll add it to the docs.

**Comment by Rendez** [comment]
Fixed, check out 1.2.1: https://github.com/qdrant/qdrant-js/releases#hd-6ac1b7a0
